ansible/roles/packages/tasks/oracle.yaml

```diff
@@ -0,0 +1,76 @@
+---
+
```

Comment:
I really don't know all the yamls in the world, and this probably doesn't really matter at all - but should there be this extra line here?